### ISSUE TITLE
Without accessibility access

### ISSUE THREAD

**Issue by BoltCitizen** [jujinkim/Frequaw-legacy#5]
I deleted the app because I am not comfortable with giving accessibility access. In another forum someone said this app should be doable without accessibility access.
